app/scrapers/football_scraper.py

```diff
@@ -0,0 +1,315 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+import bs4
+import httpx
+from bs4 import BeautifulSoup
+from config.logging import setup_logging
+from models.schemas import CalendarEvent, EventDate
+
+logger = setup_logging(__name__)
+
+
+class FootballScraper:
+    """
+    Scraper for Maccabi Tel Aviv football matches
+    """
+
+    def __init__(self):
+        self.season_link_unformatted: str = "https://www.maccabi-tlv.co.il/%d7%9e%d7%a9%d7%97%d7%a7%d7%99%d7%9d-%d7%95%d7%aa%d7%95%d7%a6%d7%90%d7%95%d7%aa/%d7%94%d7%a7%d7%91%d7%95%d7%a6%d7%94-%d7%94%d7%91%d7%95%d7%92%d7%a8%d7%aa/%d7%aa%d7%95%d7%a6%d7%90%d7%95%d7%aa/?season={season_number}#content"
+        self.upcoming_matches_url: str = "https://www.maccabi-tlv.co.il/%d7%9e%d7%a9%d7%97%d7%a7%d7%99%d7%9d-%d7%95%d7%aa%d7%95%d7%a6%d7%90%d7%95%d7%aa/%d7%94%d7%a7%d7%91%d7%95%d7%a6%d7%94-%d7%94%d7%91%d7%95%d7%92%d7%a8%d7%aa/%d7%9c%d7%95%d7%97-%d7%9e%d7%a9%d7%97%d7%a7%d7%99%d7%9d/"
+        self.seasons_links: list[str] = []
+
+    def build_seasons_links(self):
+        # Build seasons links from the website
+
+        logger.info("Building season links")
+
+        current_season_number = 75  # 2013/14
+        while (
```

Comment:
This really hard to read formatting, can we use something with wider kine length? or something more readable?

Reply:
tried my best to shorten the lines

Reply:
חושב שצריך?
זה מקה מאוד על הקריאה, לדעתי מלפני היה נוח, לא?

Reply:
הגדלתי ל-120 תווים במקום 100 כברירת מחדל, נראה סבבה. אפשר להגדיל עוד אם נרצה